docs/presets/parrot-os.md

```diff
@@ -0,0 +1,18 @@
+[Return to Presets](./README.md#parrot-os)
+
+# ParrotOS Preset
+
+This preset is inspired by the zsh look that comes with the [ParrotOS](https://www.parrotsec.org) operating system.
+It doesn't need a nerd font to be installed.
```

Comment:
This is not true. You're using the standard branch symbol (``)  in the `[git_branch]` module, which may be available in non _Nerd Fonts_, like fonts with _Powerline_ symbols enabled.

Therefore, please either change the symbol of `[git_branch]` or add the requirement for a _Nerd Font_.

Reply:
Thanks I replaced it